components/browser/search/src/main/java/mozilla/components/browser/search/provider/localization/SearchLocalizationProvider.kt

```diff
@@ -8,39 +8,6 @@ package mozilla.components.browser.search.provider.localization
  * Class providing language, country and optionally region (actual location) of the user/device to
```

Comment:
nit: As we removed the class and added an interface the docs are now outdated. Is this breaking change? If so, is it worthy to be mentioned on the changelog?

Reply:
Yeah, makes sense. I'll update that.

Reply:
👍